versions/development/SPEC.md

```diff
@@ -985,16 +992,16 @@ For example, `${true='--enable-foo' false='--disable-foo' allow_foo}` would eval
 
 Both `true` and `false` cases are required. If one case should insert no value then an empty string literal should be used, eg `${true='--enable-foo' false='' allow_foo}`
 
-> 1.  `true` and `false` values MUST be string literals.
-> 2.  `true` and `false` are only allowed if the type is `Boolean`
-> 3.  Both `true` and `false` cases are required.
-> 4.  Consider using the expression `${if allow_foo then "--enable-foo" else "--disable-foo"}` as a more readable alternative which allows full expressions (rather than string literals) for the true and false cases.
+1. `true` and `false` values MUST be string literals.
+1. `true` and `false` are only allowed if the type is `Boolean`
+1. Both `true` and `false` cases are required.
+1. Consider using the expression `${if allow_foo then "--enable-foo" else "--disable-foo"}` as a more readable alternative which allows full expressions (rather than string literals) for the true and false cases.
```

Comment:
When I've tried to do things similar to this in the past, people have pointed out that a list of `1.`s is harder to read if you're using a text editor (vs reading the rendered MD file). Personally I'm not too fussed either way so I normally let them have their way... 😄